packages/components/src/components/Table/Table.scss

```diff
@@ -11,14 +11,35 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
+@import '~@tektoncd/dashboard-components/dist/scss/vars';
+
 .tableComponent {
+  clear:both;
+
   .bx--table-toolbar {
     background: transparent;
   }
 
+  .bx--data-table-header{
+    padding-left: 0;
+    padding-bottom: 0;
+  }
+
+  .bx--data-table-container{
+    margin-top: 0;
+  }
+
   .bx--data-table thead th button.cellText.bx--table-sort {
-    padding: 0;
+    padding: 15;
```

Comment:
do we need to override the Carbon styles for a particular reason here?

Reply:
Without using this the table header looks like this... so I thought it was probably necessary to have some padding in there: 
<img width="381" alt="Screenshot 2019-12-04 at 16 37 31" src="https://user-images.githubusercontent.com/52741262/70161955-cc8df080-16b4-11ea-81c6-9dde2e2dd31d.png">

Reply:
not just the padding but the colours, border, etc.